### PR TITLE
Fix typos in architecture.md

### PR DESCRIPTION
**Pull request by yanofearth**

### CODE REVIEW

File: docs/getting-started/architecture.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ The interface handles the distribution, executiion, storage, verification and pu
...
+The primary function of the transport interface is to broadcast messages about jobs as they are created, bid upon, and executed by compute nodes.
 
 In addition to handling the distribution of messages to other nodes, the transport component is also responsible for managing the identity of individual Bacalhau nodes. This ensures that messages are properly authenticated and delivered only to authorized nodes.
 
 In addition to distributing messages, the transport component also manages the identity of individual Bacalhau nodes to ensure that messages are delivered only to authorized nodes, enhancing network security and integrity.
 
-The bprotocol point-to-point scheduling protocol that runs over [libp2p](https://libp2p.io/) transport is the primary implementation of the transport interface in production Bacalhau networks. It employs the [GossipSub](https://docs.libp2p.io/concepts/publish-subscribe/) handler to efficiently distribute job messages to other nodes on the network. This protocol enables scalable and efficient communication between nodes, ensuring messages are delivered to the appropriate nodes without congesting the network.
```
Comment (aronchick) [inline docs/getting-started/architecture.md:65]: bprotocol is what it's called :) (we should have a definition somewhere)

Comment (aronchick) [reply docs/getting-started/architecture.md:65]: @lukemarsden do we have a glossary somewhere? That might be nice.

cc @fakela @it09

Comment (fakela) [reply docs/getting-started/architecture.md:65]: We don't but we can create a page for that. Let me create an issue for glossary

Comment (fakela) [reply docs/getting-started/architecture.md:65]: Done this

**Comment by fakela** [comment]
@aronchick I added some changes

**Pull request merged**